src/test/java/racing/domain/car/CarsTest.java

```diff
@@ -0,0 +1,87 @@
+    }
+
+    private static Stream<Arguments> 중복된_이름_확인() {
+        return Stream.of(
+            Arguments.of(Arrays.asList(new Car("a", 2), new Car("a", 4), new Car("C", 1))));
+    }
+
+    private static Stream<Arguments> 자동차_개수_확인() {
+        return Stream.of(Arguments.of(Arrays.asList()));
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void Car들_중_최대_위치_값_확인(List<Car> parameterCars) {
+        final Cars cars = new Cars(parameterCars);
+        assertThat(cars.getMaxPosition()).isEqualTo(4);
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void 중복된_이름_확인(List<Car> parameterCars) {
+        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(
+            () -> new Cars(parameterCars)).withMessage("[ERROR] 중복된 이름은 들어갈 수 없습니다.");
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void 자동차_개수_확인(List<String> parameterCars) {
+        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(
+            () -> new Cars(parameterCars, 0)).withMessage("[ERROR] 자동차 개수는 적어도 1개 이상이어야 합니다.");
+    }
+
+    private static Stream<Arguments> Car들_움직임_확인() {
+        return Stream.of(
+            Arguments.of(4, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(5, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(6, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(7, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(8, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(9, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))));
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void Car들_전진_움직임_확인(int movableNumber, List<Car> parameterCars) {
+        Cars cars = new Cars(parameterCars);
+        List<Car> moveCars = new ArrayList<>();
+        for (Car car : parameterCars) {
+            moveCars.add(new Car(car.getNameValue(), car.getPositionValue() + 1));
+        }
+        assertThat(cars.driveCars(() -> movableNumber)).isEqualTo(moveCars);
+    }
+
+    private static Stream<Arguments> Car들_움직이지_않는_것_확인() {
+        return Stream.of(
+            Arguments.of(0, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(1, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(2, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
+            Arguments.of(3, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))));
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void Car들_움직이지_않는_것_확인(int movableNumber, List<Car> parameterCars) {
+        Cars cars = new Cars(parameterCars);
+        List<Car> moveCars = new ArrayList<>();
+        assertThat(cars.driveCars(() -> movableNumber)).isEqualTo(parameterCars);
+    }
```

Comment:
MethodSource를 사용하고 있는 다른 테스트들도 마찬가지로 아래와 같은 순서로 나열해주세요! 🙏
```suggestion
    @ParameterizedTest
    @MethodSource
    void Car들_움직이지_않는_것_확인(int movableNumber, List<Car> parameterCars) {
        Cars cars = new Cars(parameterCars);
        List<Car> moveCars = new ArrayList<>();
        assertThat(cars.driveCars(() -> movableNumber)).isEqualTo(parameterCars);
    }
    
    private static Stream<Arguments> Car들_움직이지_않는_것_확인() {
        return Stream.of(
            Arguments.of(0, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
            Arguments.of(1, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
            Arguments.of(2, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))),
            Arguments.of(3, Arrays.asList(new Car("a", 2), new Car("b", 4), new Car("C", 1))));
    }
```

Reply:
넵! 구현한 테스트코드 모두 말씀해주신 순서로 변경하겠습니다!